### ISSUE TITLE
table(Platform): UI is not visible, but speech output is made as ‘Checkbox’. Column count information is not conveyed.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3842]
Description: UI is not visible, but speech output is made as ‘Checkbox’. Column count information is not conveyed.

Expected: UI element shall be clearly visible when accessed.
Column/Row count information shall be conveyed on accessing the interactive UI’s within the table.

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/98775711-ac61c800-2413-11eb-8baf-41dfc193f7db.png)

**Comment by sKudum** [comment]
Single row selection shown above is present in table.Updated the title accordingly.
